src/js/models/data_filter_model.js

```diff
@@ -28,14 +28,14 @@ export class DataFilterModel extends Observable {
         points: 'points'
     }
 
-    constructor(groups, configFilters, previouslySelectedFilters, primaryGroup, childData, filterType = DataFilterModel.FILTER_TYPE.indicators) {
```

Comment:
can't we have 1 single property for `previouslySelectedFilters` and `urlFilters`? do we need them separated?